jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/upload/EditLambdaDialog.kt

```diff
@@ -280,6 +288,13 @@ class UploadToLambdaValidator {
                 )
             }
         }
+        view.memorySize.text.toIntOrNull().let {
+            if (it == null || it < MIN_MEMORY || it > MAX_MEMORY || it.rem(MEMORY_INCREMENT) != 0) {
```

Comment:
Is it too restrict for the values to be 64ish? It seems Lambda accepts non 64ish values. I guess the actual memory is the upper bound of the specified value.

Reply:
Lambda might be using the 64mb under the hood, but I've manually tested to provide a value not 64ish using AWS CLI and Lambda doesn't error out. So my point here is that it would be a bad user experience to restrict the memory while it is allowed to use non-64ish size. I don't think customers want to do the math all the time.

Reply:
+1 Slider if we can make it look nice